packs/data/pfs-season-1-bestiary.db/murderous-bathhouse-3-4.json

```diff
@@ -0,0 +1,119 @@
+            },
+            "reset": "<p>The haunt deactivates and resets after 1 minute.</p>",
+            "routine": "<p>(one action) One creature within the room must attempt a @Check[type:will|dc:23]&nbsp;save as the haunt attempts to drown them.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected and temporarily immune for 1 minute.</p>\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> A ghostly version of the creature's face appears in the water of the sink, the bath, or the wash bucket. The creature immediately experiences the sensation of being underwater, and if it can't breathe water, it must begin to hold its breath to avoid suffocation. The haunt can affect only one creature per container remaining in the haunt. If the creature leaves the room, it's immediately freed.</p>\n<p><strong>Critical Failure</strong> As failure, but the creature immediately loses [[/r 1d4 #Actions]]{1d4} actions' worth of air.</p>"
+        },
+        "saves": {
+            "fortitude": {
+                "saveDetail": "",
+                "value": 0
+            },
+            "reflex": {
+                "saveDetail": "",
+                "value": 0
+            },
+            "will": {
+                "saveDetail": "",
+                "value": 0
+            }
+        },
+        "source": {
+            "author": "",
+            "value": "Pathfinder Society Scenario #1-13: Devil at the Crossroads"
+        },
+        "statusEffects": [],
+        "traits": {
+            "di": {
+                "custom": "",
+                "selected": {},
+                "value": []
+            },
+            "dr": [],
+            "dv": [],
+            "rarity": "unique",
+            "size": {
+                "value": "med"
+            },
+            "traits": {
+                "custom": "",
+                "value": [
+                    "haunt"
+                ]
+            }
+        }
+    },
+    "img": "systems/pf2e/icons/default-icons/hazard.svg",
+    "items": [
+        {
+            "_id": "Npvqpp1yncgDSDyb",
+            "data": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Trigger</strong> A creature opens the door to the haunted chamber;</p>\n<p><strong>Effect</strong> The manifestation slams the door shut. The force deals [[/r {3d8} [bludgeoning]]]{3d8 bludgeoning damage} to any creature in or adjacent to the door's space and pushes them into the room. A creature that succeeds at a @Check[type:reflex|dc:22] save takes no damage and rolls out of the way in a random direction. On a critical success, it can choose the direction. The haunt then rolls initiative.</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Trigger</strong> A creature opens the door to the haunted chamber</p>\n<hr />\n<p><strong>Effect</strong> The manifestation slams the door shut. The force deals [[/r {3d8} [bludgeoning]]]{3d8 bludgeoning damage} to any creature in or adjacent to the door's space and pushes them into the room. A creature that succeeds at a @Check[type:reflex|dc:22] save takes no damage and rolls out of the way in a random direction. On a critical success, it can choose the direction. The haunt then rolls initiative.</p>"
```